com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -2,99 +2,101 @@
+| **- Paper White**      | The Paper White value of the display. If you do not enable **Detect Paper White**, you must specify a value here. |
 
-#### External
 
-Not supported. Mostly because every different HDR screen used to display the content would need a different LUT. It is possible to decide whether to fallback on Neutral or ACES for outputting to an HDR device.
 
-## Scripting Tonemapping settings for HDR displays
+### The HDROutputSettings API
 
-In addition to controlling the above described Volume Component parameters through the camera volume stack, it is also possible to query the [HDROutputSettings](https://docs.unity3d.com/ScriptReference/HDROutputSettings.html). This can be used for example to:
-
-- Query the display brightness limits ([min](https://docs.unity3d.com/ScriptReference/HDROutputSettings-minToneMapLuminance.htmlhttps://docs.unity3d.com/ScriptReference/HDROutputSettings-minToneMapLuminance.html) and [max](https://docs.unity3d.com/ScriptReference/HDROutputSettings-maxToneMapLuminance.html)) and [paper white value](https://docs.unity3d.com/ScriptReference/HDROutputSettings-paperWhiteNits.html).
-- [Request a change in HDR](https://docs.unity3d.com/ScriptReference/HDROutputSettings.RequestHDRModeChange.html), turning it on or off.
+The[ HDROutputSettings](https://docs.unity3d.com/ScriptReference/HDROutputSettings.html) API makes it possible to enable and disable HDR mode, as well as query certain values (such as Paper White).
 
 ## HDR Debug Views
 
-HDRP offers three debug views for HDR rendering. To access them, navigate to **Window > Analysis > Render Pipeline Debugger > Lighting > HDR**
+HDRP offers three debug views for HDR rendering. To access them, navigate to **Window > Analysis > Render Pipeline Debugger > Lighting > HDR**.
 
 #### Gamut View
 
-This debug view displays two triangles that indicate which parts of the [Rec709](https://en.wikipedia.org/wiki/Rec._709) and [Rec2020](https://en.wikipedia.org/wiki/Rec._2020) color gamuts the Scene covers.
-
-This view enables you to check color plot changes during color grading and to ensure that you are making use of the wider color gamut available in HDR.
-
 ![HDR-Output-GamutView](Images/HDR-Output-GamutView.png)
 
-#### Gamut Clip
+These triangles in this debug view indicate which parts of two specific color gamuts this Scene covers. The small triangle displays[ the Rec709](https://en.wikipedia.org/wiki/Rec._709) gamut values, and the large triangle displays the[ Rec2020](https://en.wikipedia.org/wiki/Rec._2020) gamut values. This enables you to check color plot changes while color grading. It can also help you ensure that you benefit from the wider color gamut available in HDR.
```

Comment:
Typo I missed in the Google Doc: Rec 709 link includes "the" and Rec2020 link does not include "the"